crowd-server/crowd_server/apps/food_fact_API/apps.py

```diff
@@ -0,0 +1,2 @@
+from django.apps import AppConfig
+
```

Comment:
Code missing: add config class like the bellow
<code>
 class ImageLabelConfig(AppConfig):  
 default_auto_field ='django.db.models.BigAutoField'
 name =crowd_server.apps.image_label'
</code>